extensions/applicationinsights-properties-js/Tests/Selenium/propertiesSize.tests.ts

```diff
@@ -0,0 +1,45 @@
+import { Assert, AITestClass } from "@microsoft/ai-test-framework";
+import * as pako from "pako";
+
+export class PropertiesExtensionSizeCheck extends AITestClass {
+    private readonly MAX_DEFLATE_SIZE = 15;
+    private readonly filePath = "../../dist/applicationinsights-properties-js.min.js";
```

Comment:
Probably the same for all of these check both the /dist/ which is the raw compiled version that someone *might* include vs the /browser/ version which is really the packaged version where the size is the critical issue